src/platforms/ios/iostunnel.swift

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: MIT
+// Copyright © 2018-2020 WireGuard LLC. All Rights Reserved.
```

Comment:
We should ask if we can/have to change the header. I check this. It's not a blocker.